scripts/release/packager/versions.json

```diff
@@ -0,0 +1,6 @@
+{
+	"MapboxCoreMaps": "10.0.0-beta.14.2",
```

Comment:
Should be 10.0.0-beta.15

Reply:
These values are placeholders for now. The next PR will update these based on a script.